tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,219 @@
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import delete_path_recursively, create_large_file
+
+
+# Marks
+pytestmark = [pytest.mark.tier(level=1)]
+
+# Variables
+directory_str = os.path.join(PREFIX, "testdir1")
+file_path = os.path.join(directory_str, "large_file")
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+configurations_path = os.path.join(test_data_path, "wazuh_conf.yaml")
+
+# configurations
+conf_params = {"TEST_DIRECTORIES": directory_str, "MODULE_NAME": __name__}
+parameters, metadata = generate_params(extra_params=conf_params, modes=['scheduled'])
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# Fixtures
+@pytest.fixture(scope="module", params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="function")
+def create_and_delete_file(request):
+    """Creates a file and later deletes the path."""
+
+    create_large_file(directory_str, file_path)
+    yield
+    delete_path_recursively(directory_str)
+
+
+# Tests
+@pytest.mark.parametrize("tags_to_apply", [({"ossec_conf"})])
+def test_basic_usage_modify_opened_files(tags_to_apply, get_configuration, configure_environment, 
+                                        create_and_delete_file, restart_syscheckd_function, wait_for_scan_start):
+    """
+    description: Check that files that are being scanned by syscheckd daemon
+                 can modified (renamed), and that wazuh is not
+                 blocking the files.
+
+    wazuh_min_version: 4.2
+
+    parameters:
+        - tags_to_apply:
+            type: string
+            brief: Tells which configuration to use.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - create_and_delete_file:
+            type: fixture
+            brief: Creates file to be monitored. Cleans enviroment after test.
+        - restart_syscheckd_function:
+            type: fixture
+            brief: Restart the `wazuh-syscheckd` daemon.
+        - wait for scan start:
+            type: fixture
+            brief: Wait for start of initial FIM scan start.
+
+    assertions:
+        - Verify that the file hast been modified renamed.
+        - Verify that the modificaction is done before the initial scan ends.
+    """
+
+    check_apply_test(tags_to_apply, get_configuration["tags"])
+
+    # Wait a few seconds for scan to run on created file.
+    time.sleep(3)
+
+    modify_time = None
+    # Modify the file
+    changed_path = os.path.join(directory_str, "changed_name")
```

Comment:
Could you define a new variable with the value `"changed_name"` in order to not write hardcode values?